SpatialGDK/Documentation/toc.md

```diff
@@ -1,34 +1,55 @@
-- <h3>[- SpatialOS GDK for Unreal]({{urlRoot}}/index)</h3>
-
- ***
+- <h3>SpatialOS GDK for Unreal</h3>
+- <h3>[- Welcome]({{urlRoot}}/welcome)</h3>
 - <h3>+ Get started</h3>
-    - [Installation and setup]({{urlRoot}}/setup-and-installing)
-    - [Feature list]({{urlRoot}}/features)
-    - [Toolbar]({{urlRoot}}/content/toolbar)
-    - [Glossary]({{urlRoot}}/content/glossary)
-    - [License]({{urlRoot}}/license)
+    - [Introduction]({{urlRoot}}/content/get-started/introduction)
+    - [1: Get the dependencies]({{urlRoot}}/content/get-started/dependencies)
+    - [2: Get and build the GDK’s Unreal Engine Fork]({{urlRoot}}/content/get-started/build-unreal-fork)
+    - [3: Follow the Multiserver Shooter tutorial]({{urlRoot}}/content/get-started/tutorial)
+    - Next steps - either:
+        - [Set up and run the Starter Project]({{urlRoot}}/content/get-started/gdk-and-starter-project)
+        - [Port your own project to the GDK]({{urlRoot}}/content/get-started/porting-unreal-project-to-gdk)
+- <h3>[- Feature list]({{urlRoot}}/features)</h3>
+    <!--[//]: # (TODO: Add the following from Sam at <h3> level)
+    [//]: # (- <h3>[- What's new]({{urlRoot}}/TODO)</h3>) !-->
 
  ***
-- <h3>+ Concepts</h3>
-    - [SpatialOS concepts]({{urlRoot}}/content/concepts)
-    - [Actor property handover]({{urlRoot}}/content/handover-between-server-workers)
-    - [Singleton Actors]({{urlRoot}}/content/singleton-actors)
+- <h3>Concepts and terminology</h3>
+- <h3>[- SpatialOS concepts]({{urlRoot}}/content/concepts)</h3>
+- <h3>[- Spatial Type]({{urlRoot}}/content/spatial-type)</h3>)
+    <!--[//]: # (TODO: Add the following from Mike at <h3> level)
+    [//]: # (- <h3>[- Dynamic type bindings]({{urlRoot}}/TODO)</h3>) !-->
```

Comment:
Should we just add the link now? I'll be sure to add the correct doc when it's done in gdoc.